src/Salt/UserBundle/Form/Type/SearchForm.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Salt\UserBundle\Form\Type;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormBuilderInterface;
+
+/**
+ * Class SearchForm
+ */
+class SearchForm extends AbstractType
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('organization', TextType::class, [
```

Comment:
Should be:

```
->add('organization', TextType::class, [
  'label' => 'Organization',
  'required' => false,
])
```

Try and make sure we don't have wrong indentation or extra spaces anywhere.

Also check lines 25-34 for the same issue.